README.md

```diff
@@ -1,23 +1,36 @@
 # libiio
 
+Library for interfacing with Linux IIO devices
+
+libiio is used to interface to the Linux Industrial Input/Output (IIO) Subsystem. The Linux IIO subsystem is intended to provide support for devices that in some sense are analog to digital or digital to analog converters (ADCs, DACs). This includes, but is not limited to ADCs, Accelerometers, Gyros, IMUs, Capacitance to Digital Converters (CDCs), Pressure Sensors, Color, Light and Proximity Sensors, Temperature Sensors, Magnetometers, DACs, DDS (Direct Digital Synthesis), PLLs (Phase Locked Loops), Variable/Programmable Gain Amplifiers (VGA, PGA), and RF transceivers. You can use libiio natively on an embedded Linux target (local mode), or use libiio to communicate remotely to that same target from a host Linux, Windows or MAC over USB or Ethernet or Serial.
+
+Although libiio was primarily developed by Analog Devices Inc., it is an active open source library, which many people have contributed to. It released under the GNU Lesser General Public License, version 2, this open-source license allows anyone to use the library for any purpose, on any vendors processor/FPGA/SoC, which may be controlling any vendors peripheral device (ADC, DAC, etc). This includes closed or open-source, commercial or non-commercial applications (subject to the LGPL license freedoms, obligations and restrictions).
```

Comment:
I would drop the 'for any purpose', since the license still has some restrictions (correctly mentioned later).
Also to be fully exact: 'It is released under the GNU Lesser General Public License, version 2.1 or later,'